### DISCUSSION TITLE
(Un)Collapsing the left ToC not only with the arrows

### DISCUSSION THREAD

**Discussion by dbitouze** [pradyunsg/furo#579] [Ideas] [answered]
Context
-----
- `Sphinx` 5.3.0
- `Furo` 2022.09.29

Suggestion
----
Currently, the items in the left ToC can be (un)collapsed only with their corresponding (little) arrows. It would be handier if they could be (un)collapsed by clicking on their title.

**Comment by pradyunsg** [comment, marked as answer]
That’s not possible since those are complete pages, as far as Sphinx is concerned. Since Furo is designed to expose all the information that Sphinx exposes, I don’t think it’d be sensible to make the words collapse pages since those pages would be inaccessible then, meaning that the content within them would be inaccessible as well.
